src/main/java/core/basesyntax/figure/Rectangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.figure;
+
+import base.Figure;
+
+public class Rectangle extends Figure {
+    private final double width;
+    private final double height;
+
+    public Rectangle(String color, double width, double height) {
+        super(color);
+        this.width = width;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return width * height;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: rectangle, area: " + getArea()
+                + " width: " + width + " height " + height
+                + " color: " + color);
+    }
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line at the beginning of the class definition (line 5) and the `getArea` method (line 15).